components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -451,7 +458,13 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
       newValue = nil;
     }
     if ([keyPath isEqualToString:kImageKeyPath]) {
-      tabBarItemView.iconImageView.image = newValue;
+      tabBarItemView.image = newValue;
+      [tabBarItemView invalidateIntrinsicContentSize];
```

Comment:
Can we clean this up either here or in a follow up. I see these four lines repeated 3 times here.

Reply:
Sure. Will post a follow-up PR after this merges.